aiofcm/client.py

```diff
@@ -7,9 +7,9 @@
 
 
 class FCM:
-    def __init__(self, sender_id, api_key, max_connections=10, loop=None):
-        # type: (int, str, int, Optional[asyncio.AbstractEventLoop]) -> NoReturn
-        self.pool = FCMConnectionPool(sender_id, api_key, max_connections, loop)
+    def __init__(self, sender_id, api_key, max_connections=10, max_max_connection_attempts=None, loop=None):
+        # type: (int, str, int, Optional[int], Optional[asyncio.AbstractEventLoop]) -> NoReturn
```

Comment:
Is it okay to leave type hinting style like this?
or do I have to change like
`sender_id: int, api_key: str, max_connections: int = 10, max_max_connection_attempts: Optional[int] = None, loop: Optional[asyncio.AbstractEventLoop] = None` ?

Reply:
The second option would be better!

Reply:
> The second option would be better!

I changed it in this [commit](https://github.com/Fatal1ty/aiofcm/pull/15/commits/47057d2d0c3c5b7e9046909df104eaead8bed61a), and I found there are more methods to be changed.

I opened new [PR](https://github.com/Fatal1ty/aiofcm/pull/16).

Reply:
You're awesome! Type hinting is definitely essential.